versioned_docs/version-8.0/components/modeler/desktop-modeler/flags/flags.md

```diff
@@ -68,3 +69,17 @@ To display a custom version information in the status bar of the app, configure
 ```
 
 ![Custom version info](./img/display-version.png)
+
+### Zeebe SSL certificate
+
+> ℹ️ The Modeler will read trusted certificates from your operating systems trust store.
+
+Provide an additional root certificate that the modeler should use to to validate secured connections to a Camunda Platform 8 Self-Managed installation. In case your server certificate is signed by a signing authority configure the CA certificate here, not the server certificate.
```

Comment:
```suggestion
Provide an additional root certificate that the modeler should use to validate secured connections to a Camunda Platform 8 Self-Managed installation. In case your server certificate is signed by a signing authority, configure the CA certificate here, not the server certificate.
```